src/assets/entitlements.json

```diff
@@ -0,0 +1,744 @@
+[
```

Comment:
I wonder if we need the whole file.

Reply:
not really, I can remove the other things

Reply:
done